reactive-grpc-common/src/main/java/com/salesforce/reactivegrpccommon/ReactiveGrpcGenerator.java

```diff
@@ -172,7 +171,15 @@ private String absoluteFileName(ServiceContext ctx) {
         public int methodNumber;
 
         public String methodNameUpperUnderscore() {
```

Comment:
Can you add a comment pointing to the gRPC-java code used to write this?

Reply:
Yes, good idea.